src/gui/guiChatConsole.cpp

```diff
@@ -195,7 +207,10 @@ void GUIChatConsole::draw()
 		m_screensize = screensize;
 		m_desired_height = m_desired_height_fraction * m_screensize.Y;
 		reformatConsole();
-	}
+	} else if (!m_scrollbar->getAbsolutePosition().isPointInside(core::vector2di(screensize.X, m_height)))
+		// the height of the chat window is no longer the height of the scrollbar
+		// happens while opening/closing the window
+		m_scrollbar->setRelativePosition(core::rect<s32> (screensize.X - 32, 0, screensize.X, m_height));
```

Comment:
needs to take display density and perhaps gui_scaling into account (also deduplicate it)

Reply:
moved into updateScrollbar().